src/cluster.c

```diff
@@ -1429,14 +1420,48 @@ void clusterCommandSlots(client * c) {
         /* Add cluster slots info when occur different node with start
          * or end of slot. */
         if (i == CLUSTER_SLOTS || n != getNodeBySlot(i)) {
-            addNodeReplyForClusterSlot(c, n, start, i-1);
+            addNodeReplyForClusterSlot(recording_client, n, start, i-1);
             num_masters++;
             if (i == CLUSTER_SLOTS) break;
             n = getNodeBySlot(i);
             start = i;
         }
     }
-    setDeferredArrayLen(c, slot_replylen, num_masters);
+    setDeferredArrayLen(recording_client, slot_replylen, num_masters);
+    return stopCaching(recording_client);
+}
+
+int verifyCachedClusterSlotsResponse(sds cached_response) {
+    sds generated_response = generateClusterSlotResponse();
+    int result = !sdscmp(generated_response, cached_response);
+    if (!result) serverLog(LL_NOTICE,"\ngenerated_response:\n%s\n\ncached_response:\n%s", generated_response, cached_response);
```

Comment:
It probably should be at warning if anything. This is part of the crash flow right?

Reply:
The function `verifyCachedClusterSlotsResponse` is only called at debug level using `debugServerAssertWithInfo`

Reply:
That isn't what debug level means. We have a separate debug assertion, but the log level can also be set to DEBUG. The default log level is notice, so this should be okay, but ideally it would still be LL_WARNING so that it is always printed out.

Reply:
Yes I can change this to `LL_WARNING`